index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=inter">
+    
+      
+</head>
+<body>
+    <header class="navbar">
+        <a class="welcome">Welcome</a>
+        <img class="menu" alt="button" src="Enabled.png" >
+    </header>
+    <div class="main">
+        <h1>Hey there. I’m Amalia</h1>
+        <h2>I’m a software developer</h2>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+    </div>
+    <ul class="icons">
+        <li><img src="twitter.png" alt="twitter"></li>
+        <li><img src="Linkedin.png" alt="Linkedin"></li>
+        <li><img src="medium.png" alt="medium"></li>
+        <li><img src="github.png" alt="github"></li>
+        <li><img src="angellist.png" alt="angellist"></li>    
```

Comment:
- [ ] good work using the ```li``` however these should be links, please consider using ```<a>``` tag for every list item e.g. ```<li><a href="my-site.com"><img src="angellist.png" alt="angellist"></a></li>```